### ISSUE TITLE
configure: command not found

### ISSUE THREAD

**Issue by fda77** [sudo-project/sudo#238]
Sudo 1.9.13

```
checking for posix_spawnp... yes
./configure: line 24567: ]: command not found
checking for struct dirent.d_type... yes
```

https://github.com/sudo-project/sudo/blob/main/configure#L24567

**Comment by millert** [comment]
Fixed by [defec5d](https://github.com/sudo-project/sudo/commit/defec5d46eec7345b62060049f72215ffd7f3e7e)

**Comment by fda77** [comment]
Thank you!
